### PR TITLE
Remove Msg.purged and custom managers

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/msgs/models.py [hunk 1]
```diff
@@ -199,8 +199,8 @@ class Broadcast(models.Model):
     modified_on = models.DateTimeField(auto_now=True,
                                        help_text="When this item was last modified")
 
-    purged = models.NullBooleanField(default=False,
-                                     help_text="If the messages for this broadcast have been purged")
+    purged = models.BooleanField(default=False,
```
Comment (rowanseymour) [inline temba/msgs/models.py]: `Broadcast.purged` has already been initialized in https://github.com/nyaruka/rapidpro/blob/master/temba/msgs/migrations/0040_auto_20151205_0206.py - I checked on prod servers and didn't find nulls.

File: temba/msgs/views.py [hunk 2]
```diff
@@ -342,7 +342,7 @@ class MsgActionForm(BaseActionForm):
                        ('delete', _("Delete Messages")))
 
     model = Msg
-    model_manager = 'all_messages'
+    model_manager = 'objects'
```
Comment (nicpottier) [inline temba/msgs/views.py]: Do we need this at all if it is the default?

**Comment by nicpottier** [comment]
looks good

File: temba/msgs/views.py (hunk 2 above)
Comment (rowanseymour) [inline temba/msgs/views.py]: Nope - have removed it

**Pull request merged**